src/Core/Core.csproj

```diff
@@ -52,17 +60,20 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="DataAnnotations\RequiredLocalizedAttribute.cs" />
     <Compile Include="Data\Article.cs" />
     <Compile Include="Data\ArticleComment.cs" />
-    <Compile Include="Data\BrewClubContext.cs" />
+    <Compile Include="Data\BrewClubDbContext.cs" />
```

Comment:
Changed the name because the base class is "DbContext", and I was getting confused by the other HttpContext object.